src/components/Answer.tsx

```diff
@@ -55,9 +78,12 @@ export const Answer = (props: AnswerProps) => {
     </div>
   );
 
+  const isVisuallyChecked = chosenAnswer.every(i => i === undefined) &&
+    (!incorrectAnswerId && isCorrect || isIncorrect);
```

Comment:
Some new logic here to infer what the last selected choice was so we can make it appear checked (we don't actually want to check it, or that will give us an undesired `input` state when we want them to try again with multiple attempts enabled.) After a submission, chosenAnswer will be sent as `[undefined]`, so in that state, 

- appear checked if this is a correct answer, no incorrectAnswerId present means this was the response picked, or
- appear checked if this is an incorrect answer, because only the last incorrectAnswerId is sent

Reply:
I feel like this is very confusing because the variables have not so great names. Like why is `chosenAnswer` an array? Should probably be `chosenAnswers`. If it's ordered, even better if it's called `lastChosenAnswers`.

However, I also feel it would be better if we could keep all the ids in whatever component holds the answer and that component could just set isCorrect, isChecked or whatever is needed in each Answer. That should also fix the super gnarly logic of isCorrect.

Reply:
I also don't quite get the distinction between isChecked and isVisuallyChecked. Is one of them supposed to be something like `wasEverChecked`? Or is `isVisuallyChecked` just not supposed to show up when you try a new attempt?

Reply:
Yeah there's a lot of room to refactor here, this code was extracted from Tutor without many (any?) changes so it's confusing at times... with this work I tried to operate inside the existing code as much as possible due to time. Maybe `wasChecked` is better than `isVisuallyChecked` here, or maybe I should alias the css classes so I can make the names separate, like `was-last-response` or something... I'll iterate on it and make some updates.

Reply:
Changing the css class to `answer-selected` and the var name to `isPreviousResponse` were some easy changes that I feel clarify the intent a lot more, so I've pushed those. I'll keep looking for ways to improve this tomorrow and maybe we can look at it again next week.